paper-icon-button.html

```diff
@@ -135,6 +139,23 @@
        */
       icon: {
         type: String
+      },
+
+      /**
+       * Specifies the alternate text for the button, for accessibility.
+       */
+      alt: {
+        type: String,
+        observer: "_altChanged"
+      }
+    },
+
+    _altChanged: function(newValue, oldValue) {
+      var label = this.getAttribute('aria-label');
+
+      // Don't stomp over a user-set aria-label.
+      if (!label || oldValue == label) {
```

Comment:
My hunch is that we do not care about this case. That seems like the behavior of a confused user (which may be our fault, but can be addressed elsewhere). Can you think of a scenario where it would be deliberate and desirable to 1) set `aria-label` to the same as `alt` (effectively a no-op, since `aria-label` will already be the same as `alt`) and then 2) change `alt` to something different with the expectation that `aria-label` does not change?